circle.yml

```diff
@@ -1,25 +1,48 @@
 machine:
   environment:
     # We need to set this variable to let python see the package installed through apt-get
-    PYTHONPATH: "/usr/lib/python2.7/dist-packages"
+    # PYTHONPATH: /usr/lib/python2.7/dist-packages
```

Comment:
We have to use a more recent version of scipy to get the raccoon face in some examples so we cannot use the one from the system. I will soon push an amended commit that reworks circle ci that works on my account.